MAUI/DataGrid/columns.md

```diff
@@ -0,0 +1,205 @@
+---
+layout: post
+title: Columns in .NET MAUI DataGrid control | Syncfusion
+description: Learn here all about Columns support in Syncfusion .NET MAUI DataGrid (SfDataGrid) control, its elements and more.
+platform: MAUI
+control: SfDataGrid
+documentation: UG
+---
+
+# Columns in MAUI DataGrid (SfDataGrid)
+
+The [SfDataGrid](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html) allows to create and add columns in the following two ways,

```

Comment:
The [SfDataGrid](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html) allows a user to create and add columns in the following two ways:

Reply:
Modified.